### PR TITLE
If 388 asset details notifications

### PR DESCRIPTION
**Pull request by g1glh0rn**

### CODE REVIEW

**Comment by mattmikulina** [comment]
Task linked: [IF-388 Show notifications tab in Asset view](https://app.clickup.com/t/2177764/IF-388)

File: fusionfrontend/src/app/components/content/notifications-list/notifications-list.component.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div class="notifications-list">
+  <div class="p-grid p-dir-row p-justify-between p-align-center list-header">
+    <div class="title">{{ filteredNotifications?.length | i18nPlural: titleMapping }}</div>
+    <div class="p-col"></div>
+    <div *ngIf="isInline" class="notification-state-button">
+      <button (click)="changeTab(allStates.OPEN)" [disabled]="getCurrentState() == allStates.OPEN"
```
Comment (jsylupp) [inline fusionfrontend/src/app/components/content/notifications-list/notifications-list.component.html:21]: allStates:  might NotificationState be a better name?

File: fusionfrontend/src/app/components/content/notifications-list/notifications-list.component.ts [hunk 2]
```diff
@@ -30,59 +33,43 @@ import { environment } from '../../../../environments/environment';
...
   sortField: string;
   selected: Set<ID> = new Set();
   intervalId: number;
-
   faSearch = faSearch;
   searchText = '';
   allNotifications: OispNotification[];
   filteredNotifications: OispNotification[];
+  allStates = NotificationState;
+  itemsSub: Subscription;
```
Comment (jsylupp) [inline fusionfrontend/src/app/components/content/notifications-list/notifications-list.component.ts:50]: itemsSub: is there a more specific / full name?

File: fusionfrontend/src/app/components/content/notifications-list/notifications-list.component.ts [hunk 3]
```diff
@@ -30,59 +33,43 @@ import { environment } from '../../../../environments/environment';
 })
 export class NotificationsListComponent implements OnInit, OnDestroy {
 
-  private readonly FETCHING_INTERVAL_MILLISECONDS = environment.alertFetchingIntervalSec * 1000;
-
   @Input() items$: Observable<OispNotification[]>;
```
Comment (jsylupp) [inline fusionfrontend/src/app/components/content/notifications-list/notifications-list.component.ts:36]: should we rename it to notifications?

File: fusionfrontend/src/app/store/oisp/oisp-notification/oisp-notification.service.ts [hunk 4]
```diff
@@ -59,28 +61,47 @@ export class OispNotificationService {
     return notification;
   }
 
+  getAllNotificationsUsingAlertStore(limitToDeviceId: string = null): Observable<OispNotification[]> {
```
Comment (jsylupp) [inline fusionfrontend/src/app/store/oisp/oisp-notification/oisp-notification.service.ts:64]: consider removing ...All...

**Review by jsylupp — CHANGES_REQUESTED** [review]
Looks good (just code review). Only some minor suggestions

**Pull request merged**